.changeset/nasty-owls-search.md

```diff
@@ -0,0 +1,5 @@
+---
+"@sl-design-system/website": patch
```

Comment:
For the website this changeset file is not necessary, we only need changeset for changes in the packages. This file can be removed. :)